src/coreclr/pgosupport.cmake

```diff
@@ -59,9 +56,6 @@ function(add_pgo TargetName)
                         if(HAVE_LTO)
                             target_compile_options(${TargetName} PRIVATE -flto -fprofile-instr-use=${ProfilePath} -Wno-profile-instr-out-of-date -Wno-profile-instr-unprofiled)
                             set_property(TARGET ${TargetName} APPEND_STRING PROPERTY LINK_FLAGS " -flto -fprofile-instr-use=${ProfilePath}")
-                            if(NOT LD_LLVM)
```

Comment:
Why did you remove this? I don't think the default linker supports LTO.

Reply:
lld doesn't require this condition which most builds are using. bfd, for past ~7+ years supports lto. If there are builds using bfd and requiring pgosupport, those should be covered.